tutorials/algorithms/11_gradients_framework.ipynb

```diff
@@ -0,0 +1,718 @@
+{
```

Comment:
<blockquote>QGT is a metric in geometric quantum computing, it can be regarded as a measurement of the geodesic distance of points lying on the Bloch sphere. Its real and imaginary parts give different information about the quantum state. In Qiskit, the QGT classes use the <code>Estimator</code> primitive. Their inputs are a list of circuits and parameters, and return a list of the QGTs of the expectation values of the states prepared by these circuits.</blockquote><p> </p><p>I would move this to the QGT section:</p><blockquote>The entries of the QGT for a pure state <strong>are</strong> given by [formula]</blockquote><p> </p><p>And then, under Linear Combination of Unitaries QGT, say:</p><p> </p><blockquote>This method employs a linear combination of unitaries, as explained in the Gradients section</blockquote><p> </p><p>For consistency with other formulas, I think that \psi should be \psi(\theta)), to show that the state is parametrized</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1463/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>